__tests__/app.test.js

```diff
@@ -239,10 +243,71 @@ test('400 status response with a non existent article id', () => {
       expect(body.msg).toEqual('Not found');
   }) 
 })
+
+describe('PATCH /api/articles/:article_id', () => { 
+  test('should return a status :200 and return a body with an updated increased number of votes', () => {
+    const newVote = 10;
+    const articleVoted = {inc_votes : newVote}         
+    return request(app)
+    .patch(`/api/articles/1`)
+    .send(articleVoted)
+    .expect(200)    
+    .then(({ body }) => {    
+      for (let i = 0; i < body.length; i++) {
+      expect(body[i].length).toEqual(1);      
+      expect(body[i].votes).toEqual(110);
+      }
+    });
+});
+test('should return a status :200 and return a body with an updated decreased number of votes', () => {
+  const newVote = -110;
+  const articleVoted = {inc_votes : newVote}         
+  return request(app)
+  .patch(`/api/articles/1`)
+  .send(articleVoted)
+  .expect(200)    
+  .then(({ body }) => {    
+    for (let i = 0; i < body.length; i++) {
+    expect(body[i].article_id).toEqual(1);      
+    expect(body[i].votes).toEqual(-10);
+    }
+  });
+});
+
+test("PATCH - returns a status 400 and an incorrect input message when the value of inc_votes is a format another than a number", () => {
+  const newVote = 'dog';
+  const articleVoted = {inc_votes : newVote}  
+  return request(app)
+    .patch("/api/articles/4")
+    .send(articleVoted)    
+    .expect(400)
+    .then(({ body }) => {
+      expect(body.msg).toEqual("Invalid article_id -- must be an integer");
+    });
+});
+
+
+test("PATCH - returns a status 200 and the updated article, even when there is more than one property on the request body", () => {
+  const newVote = 1;
+  const articleVoted = {inc_votes : newVote, name : 'Emad'}  
+  return request(app)
+    .patch("/api/articles/1")
+    .send(articleVoted)    
+    .expect(200)
+    .then(({ body }) => {
+      for (let i = 0; i < body.length; i++) {
+      expect(body[i]).toEqual(101);
+      }
+    });
+});
```

Comment:
Could you also please add a test that checks that a 400 is returned if the inc_votes object is incorrect, for example if it doesn't have the `inc_votes` key? eg. {not_inc_votes: 1} 🧪